packages/mainsail/source/transaction.service.ts

```diff
@@ -125,6 +127,55 @@ export class TransactionService extends Services.AbstractTransactionService {
 		);
 	}
 
+	async #createVoteFromData(
+		input: Services.VoteInput,
+		callback?: Function,
+	): Promise<Contracts.SignedTransactionData> {
+		applyCryptoConfiguration(this.#configCrypto);
```

Comment:
@shahin-hq [on this PR](https://github.com/ArdentHQ/platform-sdk/pull/53) I removed the `createTransferFromData` method since I realized it was not really neccessary (see more details on related ticket), I also moved the `if (!this.#isBooted) {` condition to the `createFromData` method, I let you know because it may apply to this action, maybe we dont need a custom method